### PR TITLE
Use PhpStanExtractor as default extractor

### PR DESCRIPTION
**Pull request by Korbeil**
Following https://github.com/symfony/symfony/pull/40457

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "description": "A pack for the Symfony serializer",
     "require": {
         "doctrine/annotations": "^1.0",
-        "phpdocumentor/reflection-docblock": "*",
+        "phpstan/phpdoc-parser": "*",
```
Comment (nicolas-grekas) [inline composer.json]: we might want to conflict with property-info < 5.4, isn't it? otherwise this might break installing <= 5.3 apps

Comment (Korbeil) [reply composer.json]: Seems fair to me, fixed :+1:

File: composer.json [hunk 2]
```diff
@@ -5,9 +5,13 @@
     "description": "A pack for the Symfony serializer",
     "require": {
         "doctrine/annotations": "^1.0",
+        "phpstan/phpdoc-parser": "*",
```
Comment (nicolas-grekas) [inline composer.json]: please use alpha order

Comment (Korbeil) [reply composer.json]: Done :+1:

**Comment by Korbeil** [comment]
since SF 5.4 & SF 6 are out, should be merged now ? cc @nicolas-grekas

**Pull request merged**